inference.py

```diff
@@ -107,6 +116,9 @@ def __init__(self, ar_ckpt, nar_ckpt, device: str = None) -> None:
         nuke_weight_norm(self.codec)
         nuke_weight_norm(self.vocos)
 
+        # Download `punkt` for sentence segmentation
+        nltk.download('punkt', quiet=True)
```

Comment:
Can you add a warning that this being downloaded? Or not keep it quiet? Having it like this seems a litte weird?

I.e. this adds NLTK as a dependency in the code, but it isn't specified anywhere. Ideally make it as an optional dependancy or add it to readme/requirements.txt dependancies.

Reply:
I have removed the quiet downloading and added `nltk` into reqs.